### PR TITLE
Added aliases

### PR DESCRIPTION
**Pull request by slaveeks**
Added aliases for:
1. `/src` - `@/`
2. `/src/application` -> `@application/`
3. `/src/presentation` -> `@presentation/`
4. `/src/infrastructure` -> `@infrastructure/`
5. `/src/domain` -> `@domain/`

### CODE REVIEW

**Comment by neSpecc** [comment]
More common way is just add '@' as alias to 'src', so urls will look better:

```
@/application/...
@/domain/...
```

For example, Nuxt has such aliases

File: tsconfig.json [hunk 1]
```diff
@@ -12,6 +13,12 @@
     "noEmit": true,
     "jsx": "preserve",
     "allowSyntheticDefaultImports": true,
+    "paths": {
+      "@/infrastructure/*": ["infrastructure/*"],
+      "@/presentation/*": ["presentation/*"],
+      "@/domain/*": ["domain/*"],
+      "@/application/*": ["application/*"]
```
Comment (neSpecc) [inline tsconfig.json:20]: did you tried this?
```suggestion
      "@/*": ["./src/*"],
```

Comment (slaveeks) [reply tsconfig.json:20]: it is not working in my IDE

Comment (slaveeks) [reply tsconfig.json:20]: but i can simplify vite.config

Comment (neSpecc) [reply tsconfig.json:20]: Try changing the `include` property:

```
"./src/**/*.ts",
"./src/**/*.vue",
```

and vite alias:
```
'@': path.resolve(__dirname, './src'),
```

Comment (slaveeks) [reply tsconfig.json:20]: `'@/': '/src/',`, i can make this in vite alias
Changing `include` property does not help

File: src/application/services/useNote.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Ref, ref } from 'vue';
-import Note from '../../domain/entities/Note';
-import { noteService } from '../../domain';
+import Note from '@/domain/entities/Note';
+import { noteService } from '@/domain/index';
```
[the remark this answers is not in the document]
Comment (slaveeks) [reply src/application/services/useNote.ts:3]: fixed

[the remark this answers is not in the document]
Comment (kseniaSs) [reply src/application/services/useNote.ts:3]: but `import { noteService } from '@/domain/index';` still exists

[the remark this answers is not in the document]
Comment (slaveeks) [reply src/application/services/useNote.ts]: fixed :)

**Review by kseniaSs — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**